### PR TITLE
changed control values to upper case

### PR DESCRIPTION
**Pull request by CWempe**
lower case did not work for me
> 2017-02-16 22:21:50.978 [WARN ] [.classic.internal.servlet.CmdServlet] - Received unknown command 'pause' for item 'HTPC_control'

Signed-off-by: Christoph Wempe <cw-git@wempe.net> (github: CWempe)

### CODE REVIEW

File: addons/binding/org.openhab.binding.kodi/README.md [hunk 1]
```diff
@@ -71,9 +71,51 @@ The Kodi thing supports the following channels:
 | shownotification        | String       | Shows the provided notification message on the screen |
 | input                   | String       | Allows to control Kodi. Valid values are: `Up`, `Down`, `Left`, `Right`, `Select`, `Back`, `Home`, `ContextMenu`, `Info`, `ShowCodec`, `ShowOSD` |
 | inputtext               | String       | This channel emulates a keyboard input |
-| systemcommand           | String       | This channel allows to send commands to shutdown/suspend/hibernate/reboot kodi |
+| systemcommand           | String       | This channel allows to send commands to `shutdown`, `suspend`, `hibernate`, `reboot` kodi |
 | mediatype               | String       | The media type of the current file. e.g. song or movie | 
 
+## Item Configuration
+demo.items
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.kodi/README.md]: please add empty lines before and after here, see https://www.eclipse.org/smarthome/documentation/development/bindings/docs.html

File: addons/binding/org.openhab.binding.kodi/README.md [hunk 2]
```diff
@@ -71,9 +71,51 @@ The Kodi thing supports the following channels:
...
+String myKodi_playuri       "PlayerURI [%s]"        { channel="kodi:kodi:myKodi:playuri" }
+String myKodi_notification  "Benachrichtigung [%s]" { channel="kodi:kodi:myKodi:shownotification" }
+String myKodi_input         "Input [%s]"            { channel="kodi:kodi:myKodi:input" }
+String myKodi_inputtext     "Inputtext [%s]"        { channel="kodi:kodi:myKodi:inputtext" }
+String myKodi_systemcommand "Systemcommand [%s]"    { channel="kodi:kodi:myKodi:systemcommand" }
+String myKodi_mediatype     "Mediatyp [%s]"         { channel="kodi:kodi:myKodi:mediatype" }
+```
+
+## Sitemap Configuration
+demo.sitemap
```
Comment (kaikreuzer) [inline addons/binding/org.openhab.binding.kodi/README.md]: same here

**Comment by CWempe** [comment]
I got two questions.
1. Is my "Signed-off-by" at the right place?
I cannot find a way to edit the original commit message.
I can only edit the PR message, add a comment or create a new commit.

2. I don't want my private mail address to be displayed here so I created a new one. Name and github hanlde are correct.
Do I have to use the address I am using for my github account?
It is not clear in the guide line. (http://docs.openhab.org/developers/contributing/contributing#sign-your-work)
Buit it looks like my "Signed-off-by" is not accepted by the eclipse "ip-validation" process in another PR.

**Comment by kaikreuzer** [comment]
> Is my "Signed-off-by" at the right place?

Yes, that's fine, thanks.

> Do I have to use the address I am using for my github account?

No, any address under which you are reachable will do.

> my "Signed-off-by" is not accepted by the eclipse "ip-validation" process in another PR.

For ESH contributions, you will have to use the e-mail address that you also use for your Eclipse account (under which you have signed the ECA).

**Pull request merged**